torch/csrc/autograd/python_torch_functions_manual.cpp

```diff
@@ -511,7 +511,7 @@ static PyObject * THPVariable_linspace(PyObject* self_, PyObject* args, PyObject
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "linspace(Scalar start, Scalar end, int64_t? steps=None, *, Tensor out=None, ScalarType dtype=None, Layout layout=torch.strided, Device device=None, bool pin_memory=False, bool requires_grad=False)",
+    "linspace(Scalar start, Scalar end, int64_t steps, *, Tensor out=None, ScalarType dtype=None, Layout layout=torch.strided, Device device=None, bool pin_memory=False, bool requires_grad=False)",
```

Comment:
Do you know why are these functions bound by hand?

Reply:
Because there's a bug in native_function.yaml generated bindings where the dtype kwarg can't be disambiguated between unset and set to the default dtype. The comment on line 525 below does a poor job of explaining this.